.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+A similar PR may already be submitted! Please search among the Pull request before creating one.
+
+Thanks for submitting a pull request! Please provide enough information so that others can review your pull request:
+
+For more information, see the CONTRIBUTING guide.
+
```

Comment:
```suggestion
<!-- A similar PR may already be submitted! Please search among the Pull request before creating one.

Thanks for submitting a pull request! Please provide enough information so that others can review your pull request.

For more information, see the CONTRIBUTING guide.
-->
```
Is it possible to comment these lines out? They're only for the author of the PR. I think it would clutter the pull request on the developers' side.

Reply:
This seems good, should I go ahead and merge?